### DISCUSSION TITLE
PostgreSql does not appear to be ruining script correctly.

### DISCUSSION THREAD

**Discussion by jtripo08** [dbeaver/dbeaver#11293] [Q&A] [answered]
#### System information: 
- window 10
- DBeaver version: 7.3.2.202101032114

#### Connection specification:
- PostGreSql 9.6
- Driver name org.postgresql.Driver

#### Describe the problem you're observing:
When I try and run the a script (an example below) I get the error SQL Error [42601]: Unterminated dollar quote started at position 3 in SQL do $$. I noticed when the editor auto selected the text if did not include the 'end $$; ' as part of the script. Even if I manually selected all the script  dbeaver would re-select and exclude the  'end $$;' (see attached image).

do $$
declare
	id1 int;
begin
	select id in id1 from user where email = 'xxxxx';
	insert
		into
		user_auth (user_id,
		auth_token,
		created_dttm,
		last_access_dttm)
	values(id1),
	'1234',
	now(),
	now());
end $$;

#### Steps to reproduce, if exist:
Enter script and execute sql script.

Image of how editor is selected script:
![image](https://user-images.githubusercontent.com/77338076/104326271-8ca05c00-54b7-11eb-8b02-da4e858d29c8.png)

**Comment by ShadelessFox** [comment, marked as answer]
Hello @jtripo08,

This is happening because you have extra ')' at line 12. If you remove it, everything will be fine.

Let me know if you issue was solved :)
